src/main/java/org/jenkinsci/test/acceptance/junit/Native.java

```diff
@@ -46,8 +47,15 @@ public void evaluate() throws Throwable {
                 private void verifyNativeCommandPresent(Native n) throws IOException, InterruptedException {
                     if (n==null)        return;
                     for (String cmd : n.value()) {
-                        if (new CommandBuilder("which",cmd).system()!=0) {
-                            throw new AssumptionViolatedException(cmd + " is needed for the test but doesn't exist in the system");
+                        if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
this tests the command is available locally - not on the machine running Jenkins which may be different.  Hey ho - this was previously the case and is likely to be the case most of the time.  A larger fix will require starting Jenkins which is the whole point of this annotation to begine with (avoids starting Jenkins if the command is not present)